registry/handlers/images.go

```diff
@@ -83,6 +91,64 @@ func (imh *imageManifestHandler) GetImageManifest(w http.ResponseWriter, r *http
 		return
 	}
 
+	supportsSchema2 := false
+	supportsManifestList := false
+	if acceptHeaders, ok := r.Header["Accept"]; ok {
```

Comment:
> Make sure to use http.CanonicalHeader when accessing the header map directly.

I agree that this is necessary in the general case, but it looks like this would be a noop if the key is already properly formatted.